src/public/js/tradeshift-api.js

```diff
@@ -1,9 +1,29 @@
+/*global Q*/
 (function() {
 	var resultArea = document.getElementById('js-api-results');
 
+	function _request(method, path, body) {
+		var host = 'http://localhost:1337/';
+		return Q.Promise(function(resolve, reject) {
+			var xhr = new XMLHttpRequest();
+			xhr.open(method, encodeURI([host, path].join('')), true);
+			xhr.setRequestHeader('Content-Type', 'application/json');
+			xhr.setRequestHeader('Accept', 'application/json');
+			xhr.onload = function() {
+				if (xhr.status === 200) {
+					resolve(xhr.responseText ? JSON.parse(xhr.responseText) : null);
+				} else {
+					reject(xhr.status);
+				}
+			};
+			xhr.send(JSON.stringify(body));
+		});
+	}
+
+
 	function getAccountInfo() {
 		var xhr = new XMLHttpRequest();
-		xhr.open('GET', encodeURI('https://a272fd4f.ngrok.io/ts/info'));
+		xhr.open('GET', encodeURI('https://a272fd4f.ngrok.io/ts/info'), true);
```

Comment:
Secure tunnel to localhost so the app can be deployed to a sandbox while running locally - https://ngrok.com/